README.md

```diff
@@ -355,9 +355,55 @@ render status: :forbidden
     Unless they have some meaning in the business domain, don't put methods in
     your model that just format your data (like code generating HTML). These
     methods are most likely going to be called from the view layer only, so their
-    place is in helpers. Keep your models for business logic and data-persistence
+    place is in helpers, if simple, or a separate class if more complex.
+    Keep your models for business logic and data-persistence
     only.
     <sup>[[link](#model-business-logic)]</sup>
+    
+    ```erb
+    <!-- bad (if complex) -->
+    <% if widget.price == 0 %>
+       Free
+    <% else %>
+       <%= widget.price * customer.discount %>
+    <% end %>
+    ```
+    
+    ```erb
+    <!-- better -->
+    <%= catalog_price(widget, customer) %>
```

Comment:
How do you feel about `widget.price_for(customer)`?

Reply:
I was aiming to indicate that a helper is a 'half-way' solution between having the view logic in the template and introducing a separate object. Adding a `price_for` method to `widget` would would mean the view logic to the model, which although might appear 'cleaner', is not something I want to encourage.